CRAMv3.tex

```diff
@@ -1410,12 +1421,11 @@ \subsubsection*{Base substitution codes (BS data series)}
 
 AN: 3
 
-and they are written as a single byte, 10 01 00 11 = 147 decimal or 0x93 in this 
-case. The whole substitution matrix is written as 5 bytes, one for each reference 
-base in the alphabetical order: A, C, G, T and N.
+The substitution matrix entry for this reference base then is written as a single byte, with the
```

Comment:
Maybe "The first byte of the substitution matrix entry for reference base A is written as a single byte, ..." and "This will then be followed by 4 more bytes representing substitutions for reference base C, G, T and N."

Reply:
Yeah, done.